pkgs/development/ocaml-modules/rpclib/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, buildDunePackage, alcotest, cmdliner, rresult, xmlm , yojson }:
+
+buildDunePackage rec {
+  pname = "rpclib";
+  version = "5.9.0";
+
+  src = fetchFromGitHub {
+    owner = "mirage";
+    repo = "ocaml-rpc";
+    rev = "v${version}";
+    sha256 = "108cw68kqzqbdbspdbdw8xrj186hrp0qz9kzslbsn0ls945641sy";
```

Comment:
Is this hash correct?

Reply:
thanks, yes, it should be `1swnnmmnkn53mxqpckdnd1j8bz0wksqznjbv0zamspxyqybmancq`